exporter/clickhouseexporter/internal/gauge_metrics.go

```diff
@@ -109,7 +109,10 @@ func (g *gaugeMetrics) insert(ctx context.Context, db *sql.DB) error {
 		}()
 
 		for _, model := range g.gaugeModels {
-			serviceName, _ := model.metadata.ResAttr.Get(conventions.AttributeServiceName)
+			var serviceName string
+			if v, ok := model.metadata.ResAttr.Get(conventions.AttributeServiceName); ok {
```

Comment:
Recommended as a helper function

Reply:
Good idea!
I've a made a helper function for it and also put some tests for this new function.
As a bonus - made some small optimizations on metrics export